test/local/senders/email.js

```diff
@@ -914,3 +914,47 @@ describe(
     )
   }
 )
+
+describe('mailer constructor:', () => {
+  let config, mailer
+
+  beforeEach(() => {
+    config = [
+      'accountSettingsUrl',
+      'accountRecoveryCodesUrl',
+      'androidUrl',
+      'initiatePasswordChangeUrl',
+      'initiatePasswordResetUrl',
+      'iosUrl',
+      'iosAdjustUrl',
+      'passwordManagerInfoUrl',
+      'passwordResetUrl',
+      'privacyUrl',
+      'reportSignInUrl',
+      'sender',
+      'sesConfigurationSet',
+      'supportUrl',
+      'syncUrl',
+      'verificationUrl',
+      'verifyLoginUrl',
+      'verifySecondaryEmailUrl',
+      'verifyPrimaryEmailUrl'
+    ].reduce((target, key) => {
+      target[key] = `mock ${key}`
+      return target
+    }, {})
+
+    return P.all([
+      require(`${ROOT_DIR}/lib/senders/translator`)(['en'], 'en'),
+      require(`${ROOT_DIR}/lib/senders/templates`).init()
+    ]).spread((translator, templates) => {
+      mailer = new Mailer(translator, templates, config)
+    })
+  })
+
+  it('set properties on self from config correctly', () => {
+    Object.entries(config).forEach(([key, expected]) => {
+      assert.equal(mailer[key], expected, `${key} was correct`)
+    })
+  })
+})
```

Comment:
This is good. But could you change the default value of sesConfigurationSet to be the empty string `''`, and add `assert.ok('sesConfigurationSet' in mailer, 'configuration key exists')` at https://github.com/mozilla/fxa-auth-server/blob/4599ef7d5629f1d9d2b5a50c4825b9380a4ddcf2/test/local/senders/email.js#L255-L271. Maybe also assert same in the test above that?